### DISCUSSION TITLE
504 Gateway Time-out

### DISCUSSION THREAD

**Discussion by bou510** [lllyasviel/Fooocus#3020] [General]
I got 504 Gateway Time-out page on Colab, is the foocus work for you guys ?

**Comment by ILikeToasters** [comment]
Might want to give more information. How are you using Fooocus? Installed locally? Colab? The more details the easier it will be to assist.

**Reply by bou510** [reply]
on Colab

**Reply by ILikeToasters** [reply]
That would be an issue with Gradio it seems. The Gradio service seems to be having issues. https://status.gradio.app/

**Reply by ashailrizvi** [reply]
yea I'm having the same issue. let's hope it comes back up soon!

**Reply by ashailrizvi** [reply]
Are there any alternatives to Gradio for running Fooocus from colab?

**Reply by mashb1t** [reply]
@ashailrizvi yes there are, e.g. cloudflared or ngrok. Check out https://github.com/lllyasviel/Fooocus/issues/2852#issuecomment-2090251628 and change the Fooocus repository URL to this one, not the fork.

**Reply by Ytwork0** [reply]
@mashb1t Should we need to copy paste same or change anything, you mention.

**Reply by mashb1t** [reply]
The most advisable thing to do is to wait, it will work again when they fixed their servers.

**Reply by Ytwork0** [reply]
@mashb1t OK , How much time its take Sir ?

**Reply by Szakbal2001** [reply]
Put this into colab

!wget https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-amd64.deb
!dpkg -i cloudflared-linux-amd64.deb
!pip install pygit2==1.12.2
!pip install torchsde==0.2.5
%cd /content
!git clone https://github.com/lllyasviel/Fooocus
https://huggingface.co/Thekingbalxd/fav_models/resolve/main/webui.py?download=true
%cd /content/Fooocus
!python entry_with_update.py --share --always-high-vram

**Reply by mashb1t** [reply]
> @mashb1t OK , How much time its take Sir ?

can't tell, it's HuggingFace/Gradio, not Fooocus which is having an issue. It should be operational again, see https://status.gradio.app/793595809#logs
